client/landing/domains/content-card/style.scss

```diff
@@ -41,7 +41,7 @@
 	}
 
 	.content-card__action-button,
-	content-card__alternate-action-button {
+	.content-card__alternate-action-button {
```

Comment:
I've tested this as well, and it's currently broken. If an alternate button is added, there is no margin below: 

<img width="753" alt="Screen Shot 2022-09-27 at 6 33 37 PM" src="https://user-images.githubusercontent.com/35543432/192667689-fb5709d8-ed60-42f6-85b8-2505bdf6507b.png">

Whereas you can see below the margin was intentional and works when just the action button is rendered with selector `.content-card__action-button`: 

<img width="743" alt="Screen Shot 2022-09-27 at 6 34 10 PM" src="https://user-images.githubusercontent.com/35543432/192667713-bdab55cf-0d7c-4487-9c34-d7f48d66d60e.png">

So if we add the period back to the class rather than removing it, we end up with this: 

<img width="771" alt="Screen Shot 2022-09-27 at 6 33 06 PM" src="https://user-images.githubusercontent.com/35543432/192667957-fb603f1d-5472-4eb7-8912-c30047a4f08e.png">

Reply:
Thank you for the calrification.